src/extension/extension.ts

```diff
@@ -61,7 +61,8 @@ export class RunmeExtension {
         return window.showErrorMessage(`Failed to start Runme server. Reason: ${e.message}`)
       }
       TelemetryReporter.sendTelemetryErrorEvent('extension.server', { data: (e as Error).message })
-      return window.showErrorMessage('Failed to start Runme server, please try to reload the window')
+      // eslint-disable-next-line max-len
+      return window.showErrorMessage(`Failed to start Runme server, please try to reload the window. Reason: ${(e as any).message}`)
```

Comment:
Would prefer not keeping eslint disable around. Happy to bump the `max-len` as well if it is set too strict.
```suggestion
      return window.showErrorMessage(
            'Failed to start Runme server, please try to reload the window. ' +
            `Reason: ${(e as any).message}`
      )
```